src/api/contributors/contributors.controller.js

```diff
@@ -5,7 +5,46 @@ module.exports = async function getContributors(req, res) {
   try {
     const response = await fetch(contribURL);
     const data = await response.json();
-    return res.json(data);
+    const finalResponse = {
+      data: data.map((contributor, key) => {
+        const {
+          name,
+          github: username,
+          image,
+        } = contributor;
+        return ({
+          type: 'contributor',
+          id: key,
+          attributes: {
+            username,
+            firstName: name.split(' ')[0],
+            lastName: name.split(' ')[1] || '',
```

Comment:
Some contributors only have their firstname or an alias. Lets just have `name` here to match the JSON coming back from the github repo. We can update the RAML accordingly.

Reply:
Change made!